test/harry/main/org/apache/cassandra/harry/ddl/SchemaSpec.java

```diff
@@ -337,7 +337,7 @@ public long[] deflateRegularColumns(Object[] regulars)
 
     public boolean isWriteTimeFromAccord()
     {
-        return transactionalMode.isPresent() && transactionalMode.get().nonSerialWritesThroughAccord;
+        return writeTimeFromAccord;
```

Comment:
I was so confused by this, then looked at the git history and looks like race condition of merge and not running CI with the most recent rebase...

Reply:
I 100% did run CI with exactly what I pushed to `cep-15-accord`. Someone else changed the history and force pushed an earlier commit.

https://github.com/aweisberg/cassandra/commit/9ba3e017629d520be625b75d88acda4eb8835224

Reply:
This also didn't cause a test failure.

Reply:
really?  I added this method because the accord harry sai test required timestamps and removing them broke things...

Reply:
Ah you are correct. I just checked and it does cause it to fail. That might even be the root failure I encountered initially, I really can't remember. I found a few things not working while I was investigating and I think I still had failures even after fixing this, but it could also be because I added tests.

Reply:
Yeah multi-node fails without further fixes, but this was the only thing breaking single node.